media/js/firefox/facebook-container-video.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
```

Comment:
Should have `$` in the parens here as the internal/sandboxed name for `window.jQuery` (passed in below).

Reply:
Added!